spring-cloud-stream-binder-kafka-docs/src/main/asciidoc/overview.adoc

```diff
@@ -271,3 +271,21 @@ Then add these dependencies at the top of the `<dependencies>` section in the po
 The versions above are provided only for the sake of the example.
 For best results, we recommend using the most recent 0.10-compatible versions of the projects.
 ====
+
+==== Excluding Kafka broker jar from the classpath of the binder based application
+If the inclusion of the Kafka broker is not desired at runtime, Kafka binder allows this particular dependency to be
+excluded from the binder so that it will not be part of the classpath. However, topic creation will be disabled from the binder
```

Comment:
Nit, but we use one sentence per line with asciidoc documentation - see http://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line